build/common/installer/scripts/tomlparser.rb

```diff
@@ -242,12 +261,20 @@ def get_command_windows(env_variable_name, env_variable_value)
   file = File.open("setenv.ps1", "w")
 
   if !file.nil?
+
+    # This path format is necessary for fluent-bit in windows
+    if @logStitchMultiline
```

Comment:
I think, this will override this  overrides  @logTailPath = "/opt/nolog*.log" which being set when the customer disabled both stderr & stdout logs.  please double check on that.

Reply:
Thanks Ganga, great catch, I will fix this